DESCRIPTION

```diff
@@ -16,4 +16,16 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.3.1
 Imports: 
-    dplyr
+    config,
+    dplyr,
+    fs,
+    janitor,
+    logger,
+    pacta.data.validation (>= 0.0.0.9000),
+    pacta.scenario.data.preparation (>= 0.0.0.9000),
+    readr,
```

Comment:
Most of these deps will be refactored into `pacta.scenario.data.preparation` in a future PR